content/docs/02-getting-started/02-nextjs-app-router.mdx

```diff
@@ -234,7 +231,7 @@ export default function Chat() {
 
 Head back to your browser (http://localhost:3000) and enter a new message. You should see a JSON object appear with the value you sent from your API route!
 
-## Introducing `ai/rsc`
+## Introducing more granular control with `ai/rsc`
```

Comment:
for reviewers: the previous subheading felt too general, the updated copy better explains what the following section is meant to explain!